timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/LeadershipContextFactory.java

```diff
@@ -113,6 +114,11 @@ public LeaderElectionHealthCheck leaderElectionHealthCheck() {
         return remoteClients().getRemoteCorruptionNotifiers();
     }
 
+    @Value.Derived
+    List<TimeLockPaxosHistoryProvider> remoteHistoryProviders() {
+        return remoteClients().getRemoteHistoryProviders();
+    }
```

Comment:
These two aren't really used in creating a leadership context, and so shouldn't be here.